packages/eas-cli/src/commands/build/index.ts

```diff
@@ -176,4 +185,21 @@ export default class Build extends EasCommand {
       message,
     };
   }
+
+  private static sanitizePlatform(flags: RawBuildFlags, platform: RequestedPlatform): void {
```

Comment:
```suggestion
  private validatePlatform(flags: RawBuildFlags, platform: RequestedPlatform): void {
```

Reply:
I think both names are incorrect this function does not validate if selected platform is correct, maybe `validateFlagsWithPlatform`?

Personally, I would create the function ensurePlatformSelected that receives BuildFlags object with optional requestedPlatform and returns BuildFlags.